sdk/tests/py/lint/lint.py

```diff
@@ -157,19 +156,19 @@ def check_regexp_line(path, f):
 
 def output_errors(errors):
     for error_type, error, line_number in errors:
-        print "%s: %s" % (error_type, error)
+        print("%s: %s" % (error_type, error))
```

Comment:
Are you using Arch Linux? These look like python3 compatibility changes.

Please split these into a different PR, and modify the shebang line to either python2 or python3.

Reply:
#2865